src/plugins/manualColumnResize/test/manualColumnResize.e2e.js

```diff
@@ -320,6 +320,44 @@ describe('manualColumnResize', () => {
     }
   });
 
+  it('should trigger an beforeColumnResize event after column size changes', () => {
+    const beforeColumnResizeCallback = jasmine.createSpy('beforeColumnResizeCallback');
+
+    handsontable({
+      data: Handsontable.helper.createSpreadsheetData(3, 3),
+      colHeaders: true,
+      manualColumnResize: true,
+      beforeColumnResize: beforeColumnResizeCallback
+    });
+
+    expect(colWidth(spec().$container, 0)).toEqual(50);
+
+    resizeColumn(0, 100);
+
+    expect(beforeColumnResizeCallback).toHaveBeenCalledWith(100, 0, false, void 0, void 0, void 0);
+    expect(colWidth(spec().$container, 0)).toEqual(100);
+  });
+
+  it('should appropriate resize colWidth after beforeColumnResize call a few times', () => {
+    const hot = handsontable({
+      data: Handsontable.helper.createSpreadsheetData(3, 3),
+      colHeaders: true,
+      manualColumnResize: true
+    });
+
+    expect(colWidth(spec().$container, 0)).toEqual(50);
+
+    hot.addHook('beforeColumnResize', () => 100);
+
+    hot.addHook('beforeColumnResize', () => 200);
+
+    hot.addHook('beforeColumnResize', () => {});
+
+    resizeColumn(0, 60);
+
+    expect(colWidth(spec().$container, 0)).toEqual(60);
```

Comment:
Shouldn't we get `200`? 🤔

Reply:
Our helper `resizeColumn` simulate `mouseup` functionality. So it uses in some sense`onMouseUp` method. This method doesn't use the value returned from `beforeColumnResize`. 
If we make use `double click` ( `afterMouseDownTimeout` method which uses the returned value from this hook) we get `200`. 
So should I change this test scenario to make use of `dbclick`?

Reply:
Ok, I see that only fake `dblclick` uses `afterMouseDownTimeout` where we can overwrite the final value. This inconsistency should be done in a separated issue.
But here, if we would like to test column/row size after dblclick then we need a test for that.

Reply:
Everything is clear. I didn't notice last commit. Now it looks ok :+1: